common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -233,20 +247,13 @@ def test_full_name_field(self):
             [u'another name', self.username],
         )
 
-        self.verify_settings_changed_events(
-            self.username, self.user_id,
+        actual_events = self.wait_for_events(event_filter=self.settings_changed_event_filter, number_of_matches=2)
+        self.assert_event_sequences_match(
```

Comment:
Ah ha! I see I'm following a similar thought-trail as Andy. I also see this pattern and think it makse sense here...